src/core/server/saved_objects/migrations/actions/calculate_exclude_filters.ts

```diff
@@ -6,13 +6,13 @@
  * Side Public License, v 1.
  */
 
-import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import type { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
```

Comment:
This has been updated in a bunch of places where we were only using 1 single type.
Also I checked the difference between importing from `typesWithBodyKey` vs `types`.
The _former_ contains a few extra `body` attributes here and there, which are deprecated, and we don't seem to be referring to them either. So using the _later_ should be fine.